packages/babel-cli/bin/babel/file.js

```diff
@@ -100,7 +100,7 @@ module.exports = function (commander, filenames, opts) {
     results = [];
 
     _.each(filenames, function (filename) {
-      if (!fs.existsSync(filename)) return;
+      if (!pathExists.sync(filename)) return;
```

Comment:
Fixed, but do I need to add path-exists to the babel-cli package.json (if it exists?).